src/templates/speciality/schemas.js

```diff
@@ -196,16 +196,9 @@ const howWeWorkSchema = {
     },
     howWeWorkWithCopy: {
       type: 'string'
-    },
-    howWeWorkWithPractises: {
-      type: 'array'
     }
```

Comment:
Why did you remove `howWeWorkWithPractises`? It still should be in the query no?

Reply:
I don't think we need it in here if its not required... 🤔 It wasn't working for me when I tried to just remove the practises in the `required` key